src/app/webApp.ts

```diff
@@ -31,6 +31,11 @@ export class PraxisIsncsciWebApp extends HTMLElement {
           display: flex;
           flex-direction: column;
         }
+
+        ::slotted(praxis-isncsci-app-layout) {
+          height: 25rem;
```

Comment:
This had to be added for the `flex-grow: 1` instruction to take effect and make the **input layout** section resize and present a scrollbar allowing the entire interface to fit inside the container and not push the classification panel out of view.